components/tag-list/tag-list.js

```diff
@@ -370,8 +371,11 @@ class TagList extends LocalizeCoreElement(InteractiveMixin(ArrowKeysMixin(LitEle
 		this._hasShownKeyboardTooltip = true;
 	}
 
-	async _handleResize(entries) {
-		this._availableWidth = Math.floor(entries[0].contentRect.width);
+	async _handleResize() {
+		this._chompIndex = 10000;
```

Comment:
This resets the container to get max available width. I haven't noticed a flicker of list items but in theory it's possible.